src/card.css

```diff
@@ -0,0 +1,113 @@
+.card {
+  max-width: 200px;
+  width: 100%;
+  box-sizing: border-box;
+  
+  display: flex;
+  flex-direction: column;
+
+  background: #fff;
+
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+
+  padding: 32px 16px 16px;
+}
+
+.card__container-image {
+  display: flex;
+  justify-content: center;
+}
```

Comment:
image for container is redundant
you can set correct size for the image and center it using margin: 0 auto